pkg/v1/cli/clientconfighelpers/windows_trusted_ca.go

```diff
@@ -10,31 +10,3546 @@ package clientconfighelpers
 //	-showcerts
```

Comment:
nit: comment to be updated and please mention the url from where we downloaded the mozilla cert bundle and a note to update this bundle periodically would be helpful.